dlrm_s_pytorch.py

```diff
@@ -488,16 +494,28 @@ def interact_features(self, x, ly):
             # li, lj = torch.tril_indices(ni, nj, offset=offset)
             # approach 2: custom
             offset = 1 if self.arch_interaction_itself else 0
-            li = torch.tensor([i for i in range(ni) for j in range(i + offset)])
-            lj = torch.tensor([j for i in range(nj) for j in range(i + offset)])
+            #li = torch.tensor([i for i in range(ni) for j in range(i + offset)])
+            #lj = torch.tensor([j for i in range(nj) for j in range(i + offset)])
```

Comment:
Expressions with two list comprehensive statements are not supported. Most simple solution is to replace those with loops but perhaps there is a better one.